README.md

```diff
@@ -72,14 +71,17 @@ The script will automatically:
 ```
 ./deploy.sh <namespace1> <namespace2> ...
 ```
-6. Port forward to APIClarity UI:
+
+Note: you can also build and deploy your own custom filter. In order to do this, please check out the [wasm-filters](https://github.com/apiclarity/wasm-filters) repo. 
```

Comment:
Note:
To build the WASM filter instead of using the pre-built binary, please follow the instructions on [wasm-filters](https://github.com/apiclarity/wasm-filters).